### DISCUSSION TITLE
Dialect needs to be explicitly supplied as of v4.0.0

### DISCUSSION THREAD

**Discussion by dangpmit** [sequelize/sequelize#15085] [Help & Questions]
Hi I want to use typescript with sequelize-cli but when I run yarn sequelize-cli db:migrate it show "Dialect needs to be explicitly supplied as of v4.0.0", anyone can give me suggestion ?
![image](https://user-images.githubusercontent.com/114461358/193733965-4a9d1540-d7b7-4d8a-a1fe-3bead5f16604.png)
![image](https://user-images.githubusercontent.com/114461358/193733975-4357d9de-9be5-4ea8-bcb3-63ff7519c97e.png)
![image](https://user-images.githubusercontent.com/114461358/193733996-47b6dfd8-ee2c-43b0-a5cc-c1e9bf1c3310.png)
![image](https://user-images.githubusercontent.com/114461358/193734005-cceea784-ba75-4e6e-9f71-aaedc51e68d4.png)
![image](https://user-images.githubusercontent.com/114461358/193734041-79c1c829-592d-4f43-85a9-a4f62ca6d4dd.png)

**Comment by krunal1325** [comment]
@padi-dev-dangpm , I'm getting the same error. did you find any solution for this?

**Reply by BilalSaeedAlam** [reply]
I have shared the solution.

**Comment by BilalSaeedAlam** [comment]
@padi-dev-dangpm & @krunal1325  here is the solution with a clean basic boilerplate. It will help you and the issue with Dialect is resolved in this. I hope this answer will be acceptable, let me know if you need any further help.
https://github.com/BilalSaeedAlam/node-typescript-sequelize

**Reply by krunal1325** [reply]
Thank you so much for your help in solving the issue! I truly appreciate the time and effort you took to assist me.

**Comment by sajidhgn** [comment]
@BilalSaeedAlam i had same problem thank you for solution
